Docs/source/running_cpp/parameters.rst

```diff
@@ -281,6 +281,14 @@ Particle initialization
       and optional argument ``<species_name>.do_symmetrize`` (whether to
       symmetrize the beam in the x and y directions).
 
+    * ``external_file``: Inject macroparticles with properties (charge, mass, position, and momentum) according to data read from external openPMD file.
+      It requires the additional arguments:
+      ``<species_name>.injection_file`` (`string`) name of the openPMD file and
+      ``<species_name>.physical_q_tot`` (`double`) optional (default is no weighting, ``weight=q_p``) when specified, WarpX injects macroparticles with the same properties as the sample of ``N`` physical particles, each with charge ``q_p``, described in the file, except their weight. Their weight is computed via ``weight=<species_name>.physical_q_tot/q_p/N``, so that the injected macroparticles represent the whole physical species.
```

Comment:
Would maybe `scale_q_tot` or `renorm_q_tot` be a more self-descriptive name for this option? Just thinking/mumbling out loud :)

Reply:
Hum, I think this would be nice to discuss with everyone in the next WarpX meeting. Because it is a style/taste/logic question. I want to try to prepare some slides for that meeting about this and I will share them with you. There I can ask for suggestions on naming and how to explain this read from external openPMD file option.

Reply:
Great idea, yes let's present this functionality and question.

Reply:
meeting summary: we just name it `q_tot` and make it optional for this profile, because that keeps it consistent with the naming in other injection styles.